.gitlab/deploy_containers/deploy_containers_a7.yml

```diff
@@ -45,6 +45,25 @@ include:
           - "-servercore"
           - "-linux"
 
+.deploy_containers-a7-base-ot:
+  extends: .docker_publish_job_definition
+  stage: deploy_containers
+  variables:
+    AGENT_REPOSITORY: agent
+    IMG_REGISTRIES: public
+  dependencies: []
+  before_script:
+    - if [[ "$VERSION" == "" ]]; then export VERSION="$(inv agent.version --major-version 7 --url-safe --pipeline-id $PARENT_PIPELINE_ID)"; fi
+    - export IMG_SOURCES="${SRC_AGENT}:v${PARENT_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-ot-beta${JMX}-amd64,${SRC_AGENT}:v${PARENT_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-ot-beta${JMX}-arm64"
+    - export IMG_DESTINATIONS="${AGENT_REPOSITORY}:${VERSION}-ot-beta${JMX}"
+
+.deploy_containers-a7-ot_external:
+  extends: .deploy_containers-a7-base-ot
+  parallel:
+    matrix:
+      - JMX:
+          - ""
+          - "-jmx"
```

Comment:
I agree with @L3n41c's assessment it seems like this could just be defined in-line without the `extends` - that seems cleaner if it's only used in one location.

Reply:
As you can read in my response, the problem is that the `deploy_containers_latest-ot` should `extend` `.deploy_containers-a7-base-ot` but it can't because gitlab currently fails to set `before_script` exported env vars in `parallel` jobs. 

I'm OK with changing it, but I'm not sure we should because the bottom line is both `deploy_containers_latest-ot` and `deploy_containers-ot` should extend `.deploy_containers-a7-base-ot`. 

I do agree on cleaning up `.deploy_containers-a7-ot-external` - that is indeed pointless.